gcovr/formats/html/write.py

```diff
@@ -146,12 +155,12 @@ def highlighter_for_file(filename):
 
 
 class PygmentHighlighting:
-    def __init__(self):
+    def __init__(self, style="default"):
```

Comment:
There is no default style.
```suggestion
    def __init__(self, style: str):
```